package.json

```diff
@@ -103,12 +97,20 @@
     "rollup-plugin-node-globals": "^1.4.0",
     "rollup-plugin-node-resolve": "^3.4.0",
     "rollup-plugin-replace": "^1.1.1",
-    "rollup-plugin-uglify": "^2.0.1",
+    "rollup-plugin-server": "^0.7.0",
+    "rollup-plugin-terser": "^6.1.0",
     "rollup-watch": "^4.0.0",
     "sinon": "^2.3.2"
   },
   "dependencies": {
     "buffer": "^5.6.0",
     "moment": "^2.25.3"
-  }
+  },
+  "homepage": "https://github.com/dropbox/dropbox-sdk-js#readme",
+  "directories": {
+    "doc": "docs",
+    "example": "examples",
+    "test": "test"
+  },
+  "license": "MIT"
```

Comment:
We might want to double check with legal / other SDKs before adding this license.

Reply:
We have this license on the package already it just isn't in the package.json